.github/workflows/dapr-release-notes.yml

```diff
@@ -7,6 +7,11 @@ name: dapr-release-notes
 
 on:
   workflow_dispatch:
+    inputs:
+      milestone_regex_suffix:
+        description: 'Suffix regex to project name'
```

Comment:
Can you make the regex "smarter", so it does not require input? The logic is as simple as picking the oldest open project that matches the regex.

Reply:
Partially matches the regex?